client/ayon_core/lib/log.py

```diff
@@ -25,49 +25,29 @@ def __init__(self, stream=None):
         self.enabled = True
 
     def enable(self):
-        """ Enable StreamHandler
+        """Enable StreamHandler
 
-            Used to silence output
+        Make StreamHandler output again
         """
         self.enabled = True
 
     def disable(self):
-        """ Disable StreamHandler
+        """Disable StreamHandler
 
-            Make StreamHandler output again
+        Used to silence output
         """
         self.enabled = False
 
     def emit(self, record):
-        if not self.enable:
+        if not self.enabled:
```

Comment:
I would move [this](https://github.com/ynput/ayon-core/pull/980/files#diff-9c91fdccab006df27620073b515952d3a831c26de66c385bda44abdf8d722036R48) condition here
```suggestion
        if not self.enabled or self.stream is None:
```